### PR TITLE
Button shape changed to circle and color changed to blue

### PR DESCRIPTION
**Pull request by Amandeep404**
Here is the demo of the changes : <br/>

### CODE REVIEW

**Comment by RahulKesharwani353** [comment]
It looks nice! can you please share a working video

File: Android-Contributions/MemeShare/app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -33,32 +33,34 @@
...
+        android:layout_width="100dp"
         android:layout_height="wrap_content"
         android:layout_margin="10dp"
         android:height="100dp"
         android:background="@drawable/circular_button"
-        android:backgroundTint="@color/teal_200"
+        android:backgroundTint="@null"
         android:padding="15dp"
         android:text="NEXT"
-        android:textSize="22sp"
```
Comment (RahulKesharwani353) [inline Android-Contributions/MemeShare/app/src/main/res/layout/activity_main.xml:46]: It is suggested to do not hard code any dimension values, 

Just create one dimen.xml file in the values folder and then use that

**Comment by RahulKesharwani353** [comment]
All the review points are suggestions for good practice only. They are not compulsory Changes.

File: Android-Contributions/MemeShare/app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (Amandeep404) [reply Android-Contributions/MemeShare/app/src/main/res/layout/activity_main.xml:46]: ok

**Comment by Amandeep404** [comment]
## Here is the demo of changes made
https://user-images.githubusercontent.com/107784525/195788265-c5ded757-045f-4e99-9e89-631323455127.mp4

**Comment by RahulKesharwani353** [comment]
Thanks for contribution

**Comment by Amandeep404** [comment]
> Thanks for contribution

Would love to if I could contribute more

**Pull request merged**